lib/3scale/backend.rb

```diff
@@ -60,8 +60,8 @@ class << self
       def new_resque_redis
         QueueStorage.connection(
           environment,
-          configuration,
-          )
+          configuration
+        )
```

Comment:
original seems alright

Reply:
IMO it looks better like this, anyway I think the indentation changes were automatic. I'll revert them.

Reply:
it's not only indentation, it is also the `,` which is nice IMO because it limits changed lines for review.

Reply:
hmm, I can't understand why this shows changed, but anyway, so be it if you want it like that.